src/boot/z_std_dma.c

```diff
@@ -9,7 +9,7 @@ u8 sDmaMgrStack[0x500];
 const char* sDmaMgrCurFileName;
 s32 sDmaMgrCurFileLine;
 
-u32 D_80009460 = 0;
+u32 gDmaMgrVerbose = 0;
```

Comment:
while longer, it might be worth being a longer name like `gDmaMgrPrintfVerbosity`
i see the existing name and think "why is the dmamgr system being verbose, for what"

Reply:
conversely, Overlay_Relocate has `gOverlayLogSeverity` which i kinda like